solutions/swb-reference/src/constants.ts

```diff
@@ -40,7 +39,6 @@ interface Constants {
   MAIN_ACCT_ALB_ARN_OUTPUT_KEY: string;
   SWB_DOMAIN_NAME_OUTPUT_KEY: string;
   MAIN_ACCT_ALB_LISTENER_ARN_OUTPUT_KEY: string;
-  ECR_REPOSITORY_NAME_OUTPUT_KEY: string;
   VPC_ID_OUTPUT_KEY: string;
   ALB_SUBNET_IDS: string[];
```

Comment:
still need ALB here?

Reply:
Yes, ALB will still be used for allowing BYON at a backend level and that feature will not be removed. Only BYON from a UI standpoint.